compiler/modules/types/core.bal

```diff
@@ -1007,7 +1011,7 @@ public function split(SemType t) returns SplitSemType  {
     }
 }
 
-public type SingleValue ()|boolean|int|float|string;
+public type SingleValue ()|boolean|int|float|decimal|string;
```

Comment:
There are 4 functions in this file that need to use this type for parameters.

Reply:
Changed.